media/js/base/site.js

```diff
@@ -12,10 +12,10 @@
             ua = ua || navigator.userAgent;
 
             if (/Win(16|9[x58]|NT( [1234]| 5\.0| [^0-9]|[^ -]|$))/.test(ua) ||
-                    /Windows ([MC]E|9[x58]|3\.1|4\.10|NT( [1234]\D| 5\.0| [^0-9]|[^ ]|$))/.test(ua) ||
+                    /Windows ([MC]E|9[x58]|3\.1|4\.10|NT( [12345]\D| 6\.0| [^0-9]|[^ ]|$))/.test(ua) ||
```

Comment:
This change is breaking downloads in IE < 9, since the class name is now `oldwin` instead of `windows` https://github.com/mozilla/bedrock/blob/master/media/js/ie8/mozilla-utils-ie8.js#L25

Thinking about it, I think changing this logic now might be a bit confusing in itself. Either we're supporting XP / Vista or we're not. We seem to want to continue allowing people to download it for now and just show a warning, so I think we should probably treat it as "supported" until the time comes when we officially want to stop people downloading it, like we do for other old systems. Right now I would read this code and assume XP/Vista should see the standard unsupported message, which isn't the case.

Reply:
Hmm, okay. Then I'll revert this change to continue using `oldwin` for XP/Vista. I'll remove the test as well, which is mentioned earlier but makes things complicated.